### ISSUE TITLE
The arrows of sidebar display wrongly in firefox. (version 2.3.7)

### ISSUE THREAD

**Issue by mensa23** [almasaeed2010/AdminLTE#1237]
![image](https://cloud.githubusercontent.com/assets/10131163/19914524/3f17d3b6-a0e6-11e6-8c31-ec9b6339fe43.png)

it works in chrome, but not works in firefox.

![image](https://cloud.githubusercontent.com/assets/10131163/19914536/5ec6f7f0-a0e6-11e6-9375-c4f5872c5acb.png)
The version we used is v2.3.7

**Comment by RikSomers** [comment]
I had the same issue yesterday, also with v2.3.7, but I use chrome instead.

**Comment by nirajannirajannirajannirajan** [comment]
i have same issue and there is also not any js errors in console

**Comment by almasaeed2010** [comment]
Hello all,
I'll look into it once I have a chance this evening. Thanks for reporting the bug!

**Comment by almasaeed2010** [comment]
Could you please provide your HTML code for the sidebar nav?

**Comment by mensa23** [comment]
Below is the part of the HTML code for the sidebar nav.

```html
<ul class="sidebar-menu">
    <li class="treeview" th:classappend="${menuConfig?.activeLeftMenu} == 'objects' ? 'active' : ''" >
        <a href="#">
            <i class="glyphicon glyphicon-th-large"></i>
                <span class="title" th:text="#{layout.menu.apps}"> Apps</span>
            <i class="fa fa-angle-left pull-right"></i>
        </a>
        <ul class="treeview-menu">
            <li th:class="${menuConfig?.activeLeftSubMenu} == 'goodcreation' ? 'active' : ''">
                <a th:href="@{/goodcreation}" href="/goodcreation"> 
                    <span class="title" th:text="#{layout.menu.objects.goodcreation}"> Good Creation </span>
                </a>
            </li>
        </ul>
    </li>
</ul>
```

**Comment by almasaeed2010** [comment]
Fixed in version v2.3.8

Thanks!

**Comment by BryantJames** [comment]
Excuse me,
 how do I change the width of the left menu bar? 
Thank you
